From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 25 Jun 2020 18:12:17 +0100
Subject: defconfig: Add Sony IMX290 image sensor driver to all Pi defconfigs

Adds the mainline IMX290 sensor driver (with extra features) to
the default configs.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 41a34ca374a8..80ff3a2d1085 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -907,6 +907,7 @@ CONFIG_VIDEO_TW2804=m
 CONFIG_VIDEO_TW9903=m
 CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
+CONFIG_VIDEO_IMX290=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index c7b9f34ceb77..60ec82bf1a11 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -921,6 +921,7 @@ CONFIG_VIDEO_TW2804=m
 CONFIG_VIDEO_TW9903=m
 CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
+CONFIG_VIDEO_IMX290=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 93fb7f03d31c..fd6e2a4422fb 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -896,6 +896,7 @@ CONFIG_VIDEO_TW2804=m
 CONFIG_VIDEO_TW9903=m
 CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
+CONFIG_VIDEO_IMX290=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 448cc2ec5c4f..641f5fe2801b 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -914,6 +914,7 @@ CONFIG_VIDEO_TW2804=m
 CONFIG_VIDEO_TW9903=m
 CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
+CONFIG_VIDEO_IMX290=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index c82e163a095c..55bf3bbbcb74 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -830,6 +830,7 @@ CONFIG_VIDEO_TW2804=m
 CONFIG_VIDEO_TW9903=m
 CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
+CONFIG_VIDEO_IMX290=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
